### DISCUSSION TITLE
Reset Button on configuration for only one node

### DISCUSSION THREAD

**Discussion by barrydouu** [zwave-js/zwave-js-ui#3130] [Q&A] [answered]
Hello

I have a zwave network with 24 devices
On one device (AEotec ZWA0122), I have a reset button, neer refresh, on the node pannel of the device (in configuration v4)
![image](https://github.com/zwave-js/zwave-js-ui/assets/46357817/befb1514-b3a2-4ae9-879f-5ba36e86f69a)

All other devices does not have this reset button.
![image](https://github.com/zwave-js/zwave-js-ui/assets/46357817/99c88167-de6c-42f8-9d72-d2c8d2dc5dfe)

Is it normal ? Why one device have a reset button, and not the other one.
I try to refresh device (and waking it up), still reset button

Perharps it's because it's v4 for one and v2 for the other (I only get one device with configuration v4)

Thank you for your help

**Comment by kpine** [comment, marked as answer]
Reset is only available starting with Configuration CC v4.

![image](https://github.com/zwave-js/zwave-js-ui/assets/791794/1782de81-2836-4fcc-974b-b1957cfe833a)

Specs: https://github.com/zwave-js/specs/blob/master/Z-Wave%20Specification%20AWG%20V1.0.pdf

**Comment by barrydouu** [comment]
Thank you for the informations
